docs/source/index.rst

```diff
@@ -40,6 +40,7 @@ model implementations and application components.
    :hidden:
 
    tutorials/audio_io_tutorial
+   tutorials/ctc_forced_alignment_api_tutorial
```

Comment:
Can you move it to Pipeline tutorial next to the existing FA tutorial, so that they are close together? Technically this tutorial might not be about pipeline, but the topic is more about FA task, so I think it's more appropriate there.